README.md

```diff
@@ -19,115 +19,141 @@ Visualization of our network architecture: [[netron]](https://netron.app/?url=ht
 2. Extract zip files under `$TRAIN_ROOT/data/widerface` as follows:
-    ```shell
-    $ tree data/widerface
-    data/widerface
-    ├── wider_face_split
-    ├── WIDER_test
-    ├── WIDER_train
-    ├── WIDER_val
-    └── trainset.json           
-    ```
-_NOTE: \
-We relabled the WIDER Face train set using [RetinaFace](https://github.com/deepinsight/insightface/tree/master/detection/RetinaFace). New labels are in 
-`$TRAIN_ROOT/data/widerface/trainset.json`, which is the COCO_format annotations file used in DALI dataloader._
+   ```shell
+   $ tree data/widerface
+   data/widerface
+   ├── wider_face_split
+   ├── WIDER_test
+   ├── WIDER_train
+   ├── WIDER_val
+   └── labelv2
+         ├── train
+         │   └── labelv2.txt
+         └── val
+             ├── gt
+             └── labelv2.txt
+   ```
+
+<!-- _NOTE: \
+We relabled the WIDER Face train set using [RetinaFace](https://github.com/deepinsight/insightface/tree/master/detection/RetinaFace). New labels are in
+`$TRAIN_ROOT/data/widerface/trainset.json`, which is the COCO_format annotations file used in DALI dataloader._ -->
 
 ## Training
+
+Following MMdetection training processing.
+
 ```Shell
-python train.py -c ./config/yufacedet.yaml -t demo 
+CUDA_VISIBLE_DEVICES=0,1 bash tools/dist_train.sh ./configs/yunet_n.py 2 12345
 ```
 
 ## Detection
+
 ```Shell
-python detect.py -c ./config/yufacedet.yaml -m weights/yunet_final.pth --target filename.jpg 
+python tools/detect_image.py ./configs/yunet_n.py ./weights/yunet_n.pth ./image.jpg
 ```
 
 ## Evaluation on WIDER Face
-1. Build NMS module.
-    ```shell
-    cd tools/widerface_eval
-    python setup.py build_ext --inplace
-    ```
 
-2. Perform evaluation. To reproduce the following performance, run on the default settings. Run `python test.py --help` for more options.
-    ```shell
-    python test.py -m weights/yunet_final.pth -c ./config/yufacedet.yaml
-    ```
+```shell
+python tools/test_widerface.py ./configs/yunet_n.py ./weights/yunet_n.pth --mode 2
+```
 
-_NOTE: We now use the Python version of `eval_tools` from [here](https://github.com/wondervictor/WiderFace-Evaluation)._
+Performance on WIDER Face (Val): confidence_threshold=0.02, nms_threshold=0.45, in origin size:
 
-Performance on WIDER Face (Val): confidence_threshold=0.3, nms_threshold=0.45, in origin size:
 ```
-AP_easy=0.882, AP_medium=0.871, AP_hard=0.767
+AP_easy=0.899, AP_medium=0.883, AP_hard=0.792
 ```
 
 ## Export CPP source code
+
 The following bash code can export a CPP file for project [libfacedetection](https://github.com/ShiqiYu/libfacedetection)
+
 ```Shell
-python exportcpp.py -c ./config/yufacedet.yaml -m weights/yunet_final.pth
+python tools/yunet2cpp.py ./configs/yunet_n.py ./weights/yunet_n.pth
 ```
 
 ## Export to onnx model
+
 Export to onnx model for [libfacedetection/example/opencv_dnn](https://github.com/ShiqiYu/libfacedetection/tree/master/example/opencv_dnn).
```

Comment:
Should also add a notice here:

***NOTICE***: This version of YuNet is newer than [the one in opencv_zoo](https://github.com/opencv/opencv_zoo/tree/master/models/face_detection_yunet). We will update the one in the opencv_zoo to keep in sync. Stay tune for the update!